frontend/src/components/misc/ErrorResult.tsx

```diff
@@ -6,7 +6,7 @@ import { capitalizeFirst } from 'utils/utils';
 import type { WrappedApiError } from '../../state/restInterfaces';
 
 interface ErrorResultProps {
-  error?: ConnectError | WrappedApiError;
+  error?: ConnectError | WrappedApiError | null;
```

Comment:
Why do we need both `undefined` and `null`?

Reply:
This was copied over from ErrorResult, sometimes the connect error defaults to null for connect query, this was clashing types.